cmd/ekco/cli/rotate-kotsadm-certs.go

```diff
@@ -0,0 +1,45 @@
+package cli
```

Comment:
I only added this for testing, but leaving it in so that if we add a button to kotsadm to rotate the certs it can exec into ekco operator pod and run this command.